config/packages/api_platform.yaml

```diff
@@ -1,15 +1,43 @@
 api_platform:
-    # The title of the API.
+    # Documentation
     title: 'DDB Cover Service'
+    description: >-
+        This service provides covers for library materials indexed by isbn, issn, faust, pid.
+        The service is provided by [Danskernes Digitale Bibliotek](https://www.danskernesdigitalebibliotek.dk/)
 
-    # The description of the API.
-    description: 'Search for covers by ISBN/FAUST/PID'
+        ### Authentication notes
 
-    # The version of the API.
+        Authentication is done via OAuth2 against auth.dbc.dk. To obtain a valid token follow instructions in
+        [1.2. Password Grant](https://github.com/DBCDK/hejmdal/blob/master/docs/oauth2.md#12-password-grant).
+        To use the "Authorize" option in this tool use your 'client_id' and 'client_secret' and fill in '@' for
```

Comment:
I should have left this for the auth PR. Let us address it there

Reply:
This text is copy paste from the custom open api spec file for v1: https://cover.dandigbib.org/api. It is "added" here because this PR refactorshow the the openapi spec is genererated.

Let's address auth issues in the auth PR